Adapters/Vungle/ISVungleAdapter/ISVungleAdapter.m

```diff
@@ -485,11 +326,6 @@ - (void)loadInterstitialInternal:(NSString *)placementId
 
     LogAdapterApi_Internal(@"placementId = %@", placementId);
 
-    // In favor of supporting all of the Mediation modes there is a need to store the Interstitial delegate
```

Comment:
Not storing the delegate might cause the adapter to fail under some of the Mediation modes